subsys/net/ip/net_context.c

```diff
@@ -229,7 +229,8 @@ int net_context_get(sa_family_t family,
 			continue;
 		}
```

Comment:
You can remove the comment `* More potential optimizations and fixes are possible` from commit message as that is always possible, no need to state the fact.

Reply:
done

Reply:
@jukkar, i found here in next_context_get,  there is similar issue  like
that 
```
/* FIXME - Figure out a way to get the correct network interface
	 * as it is not known at this point yet.
	 */
```

for `net_tcp_get`

Is it possible to move `net_tcp_get`  to `net_context_bind` where it's known that whether the iface is offloaded

Reply:
> Is it possible to move `net_tcp_get` to `net_context_bind` where it's known that whether the iface is offloaded

The `net_tcp_get()` function just setup's some TCP timers etc. So if TCP is not used before bind(), then perhaps it might be possible for listening stream socket. But for connected socket the bind() is not called at all so it would not help here.